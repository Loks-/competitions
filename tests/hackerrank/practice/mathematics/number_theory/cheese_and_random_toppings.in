6
5 2 1001
5 2 6
10 5 15
20 6 210
13 11 21
10 9 5
