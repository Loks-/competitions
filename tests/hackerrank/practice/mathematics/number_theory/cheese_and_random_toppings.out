10
4
12
120
15
0
